django_airavata/apps/workspace/static/django_airavata_workspace/js/components/experiment/ExperimentSummary.vue

```diff
@@ -357,6 +380,11 @@ export default {
         urls.navigateToEditExperiment(clonedExperiment);
       });
     },
+    cancel() {
+      services.ExperimentService.cancel({
+        lookup: this.experiment.experimentId
+      });
```

Comment:
Might be good to add a `.then(...)` and display a success message that the experiment was canceled, since it takes a while before experiments get canceled so it won't be immediately obvious that it worked or not.  See https://github.com/apache/airavata-django-portal/blob/master/django_airavata/apps/admin/static/django_airavata_admin/src/components/applications/ApplicationEditorContainer.vue#L562 for an example of displaying a success message.

Reply:
Added a then statement to display a trying to cancel message.